src/main/scala/mesosphere/marathon/health/HealthCheckWorkerActor.scala

```diff
@@ -31,9 +31,7 @@ class HealthCheckWorkerActor extends Actor with ActorLogging {
     case HealthCheckJob(task, check) =>
       val replyTo = sender() // avoids closing over the volatile sender ref
 
-      val replyWithHealth = doCheck(task, check)
-
-      replyWithHealth.onComplete {
+      doCheck(task, check).andThen {
```

Comment:
This is a little inconsistent. Here you use a `.` and in the second `andThen` you don't. Any specific reason for this?